### ISSUE TITLE
WARNING `vim.lsp.get_active_clients()` is deprecated.

### ISSUE THREAD

**Issue by igorlfs** [antosha417/nvim-lsp-file-operations#31]
Hey,

So, the plugin currently uses `vim.lsp.get_active_clients()`, which was deprecated, in favor of `vim.lsp.get_clients()`. Should be a simple change, I could make a PR

**Comment by antosha417** [comment]
prs are appreciated, thank you for your time and contribution to the project!

**Comment by rajayonin** [comment]
[Neovim 0.11 is here](https://gpanders.com/blog/whats-new-in-neovim-0-11/), this **will be deprecated in Neovim 0.12**, and it's already showing the warnings when using the plugin...

**Comment by igorlfs** [comment]
Yeah, I've been using my fork for months now. Take a look at my PR

**Comment by johannesbzd** [comment]
Neovim 0.12 has been released which has removed the feature according to :checkhealth vim.deprecated:

```
vim.lsp.get_active_clients() is deprecated. Feature was removed in Nvim 0.12
```

**Comment by tan-wei** [comment]
> Neovim 0.12 has been released which has removed the feature according to :checkhealth vim.deprecated:
> 
> ```
> vim.lsp.get_active_clients() is deprecated. Feature was removed in Nvim 0.12
> ```

Seems removed in nightly version.
